test/e2e/connectivity_test.go

```diff
@@ -121,6 +125,100 @@ func (data *TestData) runPingMesh(t *testing.T, podInfos []PodInfo, ctrname stri
 	}
 }
 
+// verifyWindowsPodConnectivity checks the Pods' connectivity after antrea-agent is restarted on Windows.
+// The Pods in the test includes both host-network Pod and the generic Pod, because CNI on Windows is also
```

Comment:
We test both the generic Pod case and the host-network Pod case, because ...

Reply:
updated.